docs/concepts/README.md

```diff
@@ -0,0 +1,217 @@
+# Duck Typing
+
+Knative uses duck typing to keep various components loosely coupled, but
+[what is duck typing](https://en.wikipedia.org/wiki/Duck_typing)? It is
+reasoning about a resource's control plane shape and behaviors based on a common
+definition of that contact. If a resource has the same fields in the same schema
```

Comment:
```suggestion
Knative enables [loose coupling](https://en.wikipedia.org/wiki/Loose_coupling) of its components by using [duck typing](https://en.wikipedia.org/wiki/Duck_typing). Duck typing means that the compatibility of a resource for use in a Knative system is determined by certain properties being present that can be used to identify the resource control plane shape and behaviors. These properties are based on a set of common definitions for different types of resources, called duck types. If a resource has the same fields in the same schema
```

Reply:
You don't have to use this but I suggested some stuff that made this make more sense to me 🙂

Reply:
the whole point I am drafting it and showing it :D make it make sense to others...